database/seed-data/fakeDataGenerator.js

```diff
@@ -1,23 +1,16 @@
 const faker = require('faker');
 
-let fakeHeader = () => {
-  return (faker.lorem.sentence().slice(0, -1));
-};
-
-let fakeSentence = () => {
-  return (faker.lorem.sentence());
-};
-
-let fakeDescription = () => {
-  return (faker.lorem.sentences());
-};
-
-let fakeTitle = () => {
-  return (faker.name.jobArea());
-};
+let fakeHeader = () => (faker.lorem.sentence().slice(0, -1));
+let fakeSentence = () => (faker.lorem.sentence());
+let fakeDescription = () => (faker.lorem.sentences());
+let fakeTitle = () => (faker.name.jobArea());
```

Comment:
Nice! Saved over 10 lines with those one-liners. I am wondering. When you don't use curlies in an arrow function then you get an implied return, no? So I believe that there's no need to wrap your function body e.g. faker.name.jobArea() in parentheses.

Reply:
Thanks for pointing that out. I will clean it up further!